101-machine-learning-compute-create-computeinstance/azuredeploy.json

```diff
@@ -0,0 +1,106 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {        
+        "workspaceName": {
+            "type": "string",
+            "metadata": {
+              "description": "Specifies the name of the Azure Machine Learning workspace which will hold this compute taget."
+            }
+        },
+        "computeName": {
+            "type": "string",
+            "metadata": {
+              "description": "Specifies the name of the Azure Machine Learning compute target."
+            }
+        },        
+        "location": {
+            "type": "string",
+            "defaultValue":"[resourceGroup().location]",
+            "metadata": {
+              "description": "Location of the Azure Machine Learning workspace."
+            }
+        },
+        "adminUserName": {
+            "defaultValue": "",
+            "type": "securestring",
+            "metadata": {
+              "description": "The admin username needed to access the virtual machine."
+            }
+        },
+        "adminPublicKey": {
+            "defaultValue": "",
+            "type": "securestring",
+            "metadata": {
+              "description": "The SSH public key of the administrator user account."
+            }
+        },
+        "sshPublicAccess": {
```

Comment:
what value is expected here?

Reply:
Enabled. Disabled. I will add that to the description.